core/commands/storage.go

```diff
@@ -1569,13 +1569,13 @@ This command updates host information and broadcasts to the BTFS network.
 
 Examples
 
-To set the max price per GiB to 1 BTT:
+To set the min price per GiB to 1000000 JUST (1 BTT):
 $ btfs storage announce --host-storage-price=1`,
```

Comment:
i believe this should be 1000000 since we are trying to show how to set to 1 BTT.

Reply:
@richlee33 Thanks for the catch. I've updated the sample command. Please let me know how it looks. Thx